README.md

```diff
@@ -2,27 +2,42 @@
 
 A library and command line interface for the CVE Project services.
 
+## Reequirements
+
+- Python version 3.6 or greater
+- pip
+
 ## Installation
 
+Universal Installation that should work on Linux, MacOS and Windows.
```

Comment:
Is this true for Windows? Looking at https://packaging.python.org/tutorials/installing-packages/#installing-from-pypi, the instructions include `py -m` before the `pip install`. Maybe we could link to this URL below the `pip install` command with something like: 

> For more information on installing Python packages from PyPI, see the [Python Packaging User Guide](https://packaging.python.org/tutorials/installing-packages/#installing-from-pypi).

Reply:
You have to have pip installed on windows, then it should be the case. If you don't, you would do it as you say.